hoomd/conftest.py

```diff
@@ -87,3 +136,1266 @@ def pytest_sessionfinish(session, exitstatus):
+#                       'centers': "invalid",
+#                       'ignore_statistics': 0},
+#                      {'diameters': 2,
+#                       'centers': [(0, 0, 0),
+#                                   (1, 1, 1),
+#                                   (1, 0, 1),
+#                                   (0, 1, 1),
+#                                   (1, 1, 0),
+#                                   (0, 0, 1),
+#                                   (2, 2, 1),
+#                                   (3, 5, 3)],
+#                       'ignore_statistics': 1},
+#                      {'diameters': [1, 4, 2, 8, 5],
+#                       'centers': 4,
+#                       'ignore_statistics': 0}]
+#         return args_list
+#     return get_args
+
+
+# @pytest.fixture(scope="session")
+# def shape_dict_conversion_args(convex_polygon_parameters,
+#                                convex_polygon_valid_args,
+#                                convex_polyhedron_parameters,
+#                                convex_polyhedron_valid_args,
+#                                ellipsoid_parameters,
+#                                ellipsoid_valid_args,
+#                                faceted_ellipsoid_parameters,
+#                                faceted_ellipsoid_valid_args,
+#                                faceted_ellipsoid_union_parameters,
+#                                faceted_ellipsoid_union_valid_args,
+#                                sphere_parameters,
+#                                sphere_valid_args,
+#                                sphere_union_parameters,
+#                                sphere_union_valid_args,
+#                                convex_spheropolyhedron_union_parameters,
+#                                convex_spheropolyhedron_union_valid_args,
+#                                sphinx_parameters,
+#                                sphinx_valid_args):
+#     def get_valid_args():
+#         return [(convex_polygon_parameters(),
+#                  convex_polygon_valid_args()),
+#                 (convex_polyhedron_parameters(),
+#                  convex_polyhedron_valid_args()),
+#                 (ellipsoid_parameters(),
+#                  ellipsoid_valid_args()),
+#                 (faceted_ellipsoid_parameters(),
+#                  faceted_ellipsoid_valid_args()),
+#                 (faceted_ellipsoid_union_parameters(),
+#                  faceted_ellipsoid_union_valid_args()),
+#                 (sphere_parameters(),
+#                  sphere_valid_args()),
+#                 (sphere_union_parameters(),
+#                  sphere_union_valid_args()),
+#                 (convex_spheropolyhedron_union_parameters(),
+#                  convex_spheropolyhedron_union_valid_args()),
+#                 (sphinx_parameters(),
+#                  sphinx_valid_args())]
+#     return get_valid_args
+
+
+# @pytest.fixture(scope="session")
+# def integrator_args(convex_polygon_integrator,
```

Comment:
Let's not add thousands of lines of commented code.

Reply:
Yeah, this can all be safely removed if the fixtures were added to a new file which they seem to be.